app/views/content/is-teaching-right-for-me/qualifications-you-need-to-teach.md

```diff
@@ -40,7 +40,7 @@ Talk to your teacher training provider or [find a course](https://www.find-postg
 
 If you do not already have an undergraduate degree, you can train to be a teacher as part of your bachelor's degree and get QTS.
 
-Or you may be eligible to apply for a new [teacher degree apprenticeship (TDA)](https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/). This means you'll work in a school and earn a salary while getting a bachelor’s degree and QTS. So you will not have to pay tuition fees.
+Or you may be eligible to apply for a new [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teaching-degree-apprenticeships). This means you'll work in a school and earn a salary while getting a bachelor’s degree and QTS. So you will not have to pay tuition fees.
 
 Applications for the new TDAs are expected to open in autumn 2024 for both primary and secondary, with training due to start in autumn 2025.
```

Comment:
```suggestion
Applications for the new TDAs are expected to open in autumn 2024 for both primary and secondary, with training due to start in autumn 2025.
```
I think we should probably delete this sentence here - that info is on the TDA page